app/parse.py

```diff
@@ -15,8 +31,126 @@ class Product:
     num_of_reviews: int
 
 
+def init_driver() -> webdriver.Chrome:
+    options = Options()
+    options.add_argument("--headless")
+    options.add_argument("--no-sandbox")
+    options.add_argument("--disable-dev-shm-usage")
+
+    driver = webdriver.Chrome(
+        service=ChromeService(ChromeDriverManager().install()), options=options
+    )
+    return driver
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        accept_button = WebDriverWait(driver, 2).until(
+            ec.element_to_be_clickable((By.CLASS_NAME, "acceptCookies"))
+        )
+        accept_button.click()
+        print("Cookies accepted")
+    except Exception as e:
+        print(f"Cookies not accepted: {e}")
+
+
+def load_all_products(driver: webdriver.Chrome) -> None:
+    while True:
+        try:
+            more_button = WebDriverWait(driver, 2).until(
+                ec.presence_of_element_located(
+                    (By.CLASS_NAME, "ecomerce-items-scroll-more")
+                )
+            )
+
+            if (
+                "disabled" not in more_button.get_attribute("class")
+                and more_button.is_displayed()
+            ):
+                more_button.click()
+                print("Clicked 'More' button")
+            else:
+                print("'More' button is either disabled or not displayed")
+                break
+        except Exception as e:
+            print("The 'More' button was not found or became unavailable:", e)
```

Comment:
Similar to the previous comment, catching a general `Exception` here can obscure the actual issue. It's better to catch specific exceptions related to Selenium operations, such as `NoSuchElementException` or `ElementNotInteractableException`.